pyodide_build/tests/recipe/test_builder.py

```diff
@@ -248,11 +248,23 @@ def _check_patches_extra(self) -> Self:
         return self
 
 
-def test_needs_rebuild(tmpdir):
+@pytest.mark.parametrize("is_wheel", [False, True])
+def test_needs_rebuild(tmpdir, is_wheel):
     pkg_root = Path(tmpdir)
     buildpath = pkg_root / "build"
     meta_yaml = pkg_root / "meta.yaml"
-    packaged = buildpath / ".packaged"
+    version = "12"
+    if is_wheel:
+        dist_dir = pkg_root / "dist"
+        dist_dir.mkdir()
+        # Build of current version with wrong abi
+        (dist_dir / "regex-12-cp311-cp311-pyodide_2024_0_wasm32.whl").touch()
+        # Build of old version with current abi
+        (dist_dir / "regex-11-cp312-cp312-pyodide_2024_0_wasm32.whl").touch()
+        # the version we're trying to build
+        packaged = dist_dir / "regex-12-cp312-cp312-pyodide_2024_0_wasm32.whl"
```

Comment:
```suggestion
        # Build of current version with wrong abi
        (dist_dir / "regex-12-cp311-cp311-pyodide_2024_0_wasm32.whl").touch()
        # Build of old version with current abi
        (dist_dir / "regex-11-cp312-cp312-pyodide_2025_0_wasm32.whl").touch()
        # the version we're trying to build
        packaged = dist_dir / "regex-12-cp312-cp312-pyodide_2024_0_wasm32.whl"
```

I think this might have been a typo? Please feel free to discard otherwise.

Reply:
I think this test shims the abi tag so the "current" abi is actually whatever is specified here. In any case, the test passes as written.